verisignmdns/src/test/java/denominator/verisignmdns/VrsnMDNSTest.java

```diff
@@ -0,0 +1,340 @@
+            + "</ns2:getResourceRecordListRes>" + "</S:Body>" + "</S:Envelope>";
+
+    public static final String rrListCNAMETypesTemplete = TEMPLATE_HEAD + "<urn2:getResourceRecord>"
+            + "<urn2:resourceRecordId>" + RESOURCE_RECORD_ID + "</urn2:resourceRecordId>" + "</urn2:getResourceRecord>"
+            + TEMPLATE_TAIL;
+
+    public static final String getrrListCNAMETypesTemplete = TEMPLATE_HEAD
+            + "<ns3:getResourceRecordList xmlns=\"urn:com:verisign:dnsa:auth:schema:1\" xmlns:ns2=\"urn:com:verisign:dnsa:messaging:schema:1\" xmlns:ns3=\"urn:com:verisign:dnsa:api:schema:1\">"
+            + "<ns3:domainName>" + VALID_ZONE_NAME1 + "</ns3:domainName>" + "<ns3:resourceRecordType>" + VALID_RR_TYPE1
+            + "</ns3:resourceRecordType>" + "</ns3:getResourceRecordList>" + TEMPLATE_TAIL;
+
+    public static final String nAPTRDataResponse = "<?xml version='1.0' encoding='UTF-8'?>"
+            + "<S:Envelope xmlns:S='http://www.w3.org/2003/05/soap-envelope'>"
+            + "<S:Body>"
+            + "<ns2:getResourceRecordListRes xmlns='urn:com:verisign:dnsa:api:schema:2' xmlns:ns2='urn:com:verisign:dnsa:api:schema:1' xmlns:ns3='urn:com:verisign:dnsa:auth:schema:1' xmlns:ns4='urn:com:verisign:dnsa:messaging:schema:1'>"
+            + "<ns2:callSuccess>true</ns2:callSuccess>" + "<ns2:totalCount>2</ns2:totalCount>" + "<ns2:resourceRecord>"
+            + "<ns2:resourceRecordId>19076156</ns2:resourceRecordId>" + "<ns2:owner>"
+            + VALID_OWNER1
+            + "</ns2:owner>"
+            + "<ns2:type>"
+            + VALID_RR_TYPE3
+            + "</ns2:type>"
+            + "<ns2:ttl>"
+            + VALID_TTL1
+            + "</ns2:ttl>"
+            + "<ns2:rData>"
+            + VALID_RData_NAPTR
+            + "</ns2:rData>"
+            + "</ns2:resourceRecord>"
+            + "<ns2:resourceRecord>"
+            + "<ns2:resourceRecordId>19049261</ns2:resourceRecordId>"
+            + "<ns2:owner>"
+            + VALID_OWNER2
+            + "</ns2:owner>"
+            + "<ns2:type>"
+            + VALID_RR_TYPE3
+            + "</ns2:type>"
+            + "<ns2:ttl>"
+            + VALID_TTL1
+            + "</ns2:ttl>"
+            + "<ns2:rData>"
+            + VALID_RData_NAPTR
+            + "</ns2:rData>"
+            + "</ns2:resourceRecord>"
+            + "</ns2:getResourceRecordListRes>" + "</S:Body>" + "</S:Envelope>";
+
+    public static final String zoneListRequestTemplate = TEMPLATE_HEAD
+            + "<ns3:getZoneList xmlns='urn:com:verisign:dnsa:messaging:schema:1' xmlns:ns2='urn:com:verisign:dnsa:auth:schema:1' xmlns:ns3='urn:com:verisign:dnsa:api:schema:1'>"
+            + "<ns3:listPagingInfo><ns3:pageNumber>1</ns3:pageNumber><ns3:pageSize>500</ns3:pageSize>"
+            + "</ns3:listPagingInfo>" + "</ns3:getZoneList>" + TEMPLATE_TAIL;
+
+    public static final String zoneListResponse = "<?xml version='1.0' encoding='UTF-8'?>"
```

Comment:
hard to know where to start with this file. If you are going to use 10+line xml literals, format them so that they are legible.